src/libs/Network/SequentialQueue.js

```diff
@@ -62,6 +67,16 @@ function process() {
     return currentRequest;
 }
 
+/**
+ * Gets the current Onyx queued updates, apply them and clear the queue if the queue is not paused.
+ */
+function flushOnyxUpdatesQueue() {
+    if (isQueuePaused) {
```

Comment:
This could use a comment to explain why it's OK to skip the flushing when the queue is paused.

Reply:
Added